logging/logger.go

```diff
@@ -34,11 +34,24 @@ func Set(level string, textLogging bool) error {
 		return err
 	}
 
+	fm := logrus.FieldMap{
+		logrus.FieldKeyTime: "@timestamp",
+		logrus.FieldKeyMsg:  "message",
+	}
+
 	logger := logrus.New()
 	if textLogging {
-		logger.Formatter = &logrus.TextFormatter{}
+		logger.Formatter = &logrus.TextFormatter{
+			TimestampFormat: time.RFC3339Nano,
+			FieldMap:        fm,
+		}
 	} else {
-		logger.Formatter = logrustash.DefaultFormatter(logrus.Fields{})
+		logger.Formatter = logrustash.LogstashFormatter{
```

Comment:
Not using the default formatter means that we lose both of the default fields specified there: `@version` and `type`.
Adding `Fields: logrustash.logstashFields` to the formatter should fix this.